tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -54,29 +58,102 @@ struct SynthOrdered {
     }
 };
 
-template <class T>
-inline constexpr bool is_pair = false;
-template <class A, class B>
-inline constexpr bool is_pair<std::pair<A, B>> = true; // TRANSITION, std::pair spaceship not yet implemented
+struct OrderedChar {
+    OrderedChar() = default;
+    OrderedChar(const char c) : c(c){};
 
-template <class Container>
-void ordered_containers_test(const Container& smaller, const Container& smaller_equal, const Container& larger) {
+    OrderedChar& operator=(const char& other) {
+        c = other;
+        return *this;
+    }
+
+    operator char() const {
+        return c;
+    }
+
+    char c;
+};
+
+auto operator<=>(const OrderedChar& left, const OrderedChar& right) {
+    return left.c <=> right.c;
+}
+
+auto operator==(const OrderedChar& left, const OrderedChar& right) {
+    return left.c == right.c;
+}

```

Comment:
No change requested, but I note that you could save a few lines here by putting:
```c++
auto operator<=>(const OrderedChar&) const = default;
```
or
```c++
friend auto operator<=>(const OrderedChar&, const OrderedChar&) = default;
```
in the class body and dropping these definitions. These do what the defaulted definitions would, and a class with a defaulted `<=>` gets a defaulted `==` implicitly ([class.compare.default]/5).